test/integration/clientset_test.go

```diff
@@ -84,77 +85,81 @@ func TestGroupVersion(t *testing.T) {
 // TestNoName checks that all creates fail for objects that have no
 // name given.
 func TestNoName(t *testing.T) {
-	client, shutdownServer := getFreshApiserverAndClient(t)
+	clientset, shutdownServer := getFreshApiserverAndClient(t)
 	defer shutdownServer()
-	scClient := client.Servicecatalog()
+	client := clientset.Servicecatalog()
 
 	ns := "namespace"
 
-	if br, e := scClient.Brokers().Create(&v1alpha1.Broker{}); nil == e {
+	if br, e := client.Brokers().Create(&v1alpha1.Broker{}); nil == e {
 		t.Fatal("needs a name", br.Name)
 	}
-	if sc, e := scClient.ServiceClasses().Create(&v1alpha1.ServiceClass{}); nil == e {
+	if sc, e := client.ServiceClasses().Create(&v1alpha1.ServiceClass{}); nil == e {
 		t.Fatal("needs a name", sc.Name)
 	}
-	if i, e := scClient.Instances(ns).Create(&v1alpha1.Instance{}); nil == e {
+	if i, e := client.Instances(ns).Create(&v1alpha1.Instance{}); nil == e {
 		t.Fatal("needs a name", i.Name)
 	}
-	if bi, e := scClient.Bindings(ns).Create(&v1alpha1.Binding{}); nil == e {
+	if bi, e := client.Bindings(ns).Create(&v1alpha1.Binding{}); nil == e {
 		t.Fatal("needs a name", bi.Name)
 	}
 }
 
+// TestBrokerClient exercises the Broker client.
 func TestBrokerClient(t *testing.T) {
-	client, shutdownServer := getFreshApiserverAndClient(t)
+	var (
+		clientset, shutdownServer = getFreshApiserverAndClient(t)
+		client                    = clientset.Servicecatalog().Brokers()
```

Comment:
I changed the names throughout because the longer names are about 5x harder to read for my eyes.